src/pages/home.tsx

```diff
@@ -0,0 +1,142 @@
+import React from 'react';
+import styled from 'styled-components';
+import imgItem1 from 'src/assets/img/imgItem1.svg';
+import section2Bg from 'src/assets/img/section2_bg.svg';
+import section2img from 'src/assets/img/section2img.png';
+import homeContent from '../../content/pages/home.yml';
+
+const Main = styled.main`
+  background-color: rgb(244, 247, 249);
+`;
+
+const FirstSection = styled.section`
+  /* padding: 6.5em 0 0 8.4em; */
+  display: flex;
+  justify-content: flex-end;
+  align-items: center;
+  min-height: 100vh;
+  /* flex-wrap: wrap; */
+`;
+const LeftSideContent = styled.div`
+  /* padding-left: 8em;
+  padding-top: 10em; */
+  padding-left: 5rem;
+  flex-basis: 50%;
+  flex-grow: 1;
+`;
+
+const Title = styled.h1`
+  font-size: 50px;
+  font-weight: bold;
+  line-height: 1.2em;
+  margin-top: 1.5em;
```

Comment:
I'd suggest to create separate components for each typography element (can be in one file) to put styles only in one place and align it strictly with Figma file:
![image](https://user-images.githubusercontent.com/28801003/146416161-ca49b95f-29cf-4739-98ff-54dbc8eb80d1.png)